geoviews/plotting/mpl/__init__.py

```diff
@@ -268,9 +267,10 @@ class GeoRGBPlot(GeoImagePlot):
     def get_data(self, element, ranges, style):
         self._norm_kwargs(element, ranges, style, element.vdims[0])
         style.pop('interpolation', None)
-        zs = get_raster_array(element)[::-1]
+        zs = get_raster_array(element)
```

Comment:
I agree this seems a bit weird. Can you try to see if things align with an `ImageStack` with one layer and a normal `Image`?

Reply:
Without the `[::-1]`
```python
import numpy as np
import geoviews as gv
import holoviews as hv
import cartopy.crs as ccrs

hv.extension("matplotlib")

x = np.arange(-120, -117)
y = np.arange(40, 43)
a = np.array([[1, np.nan, 1], [np.nan, np.nan, 1], [np.nan] * 3])

img_stack = gv.ImageStack(
    (x, y, a), kdims=["x", "y"], vdims=["a"], crs=ccrs.PlateCarree()
)
img = gv.Image((x, y, a), kdims=["x", "y"], vdims=["a"], crs=ccrs.PlateCarree())

img_stack + img
```

<img width="861" alt="image" src="https://github.com/holoviz/geoviews/assets/15331990/de736098-115f-49d6-a19c-45a97dc2b895">

Reply:
With [::-1]

<img width="834" alt="image" src="https://github.com/holoviz/geoviews/assets/15331990/2f5d0c25-ea30-4421-bb25-1f842d5379db">

Reply:
With the [::-1] dropped, everything is consistent:
```python
import numpy as np
import geoviews as gv
import holoviews as hv
import cartopy.crs as ccrs

hv.extension("matplotlib")

x = np.arange(-120, -117)
y = np.arange(40, 43)
a = np.array([[1, np.nan, 1], [np.nan, np.nan, 1], [np.nan] * 3])

img_stack = gv.ImageStack(
    (x, y, a), kdims=["x", "y"], vdims=["a"], crs=ccrs.PlateCarree()
)
img = gv.Image((x, y, a), kdims=["x", "y"], vdims=["a"], crs=ccrs.PlateCarree())
img_stack_hv = hv.ImageStack((x, y, a), kdims=["x", "y"], vdims=["a"])
img_hv = hv.Image((x, y, a), kdims=["x", "y"], vdims=["a"])

img_stack + img + img_stack_hv + img_hv
```